metaflow/includefile.py

```diff
@@ -17,11 +20,41 @@ def __init__(self, logger, is_text, encoding, path):
         self._is_text = is_text
         self._encoding = encoding
         self._path = path
-        self._size = os.path.getsize(self._path)
+        self._size = 0
 
-    def __call__(self):
+    @classmethod
+    def is_file_handled(cls, path):
```

Comment:
I find the return tuple slightly uncommon for UX. By the function name I typically expect it to return a boolean. WDYT?

If you agree, one alternative would be to accept a error_msg as an inout arg and you can set it if a valid object is passed in.
I don't know if its easy to differentiate between the following cases:
```
Assume, is_file_handled(cls, path, error_msg=None).

Case 1:
is_file_handled(path)

Case 2:
error_msg = None
is_file_handled(path, error_msg)

In C++ land a valid lval is specified in the second case.
```